CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking Changes
 - Modify `Signer`'s `signTransaction` method to return an array of signed bytes rather than a string based signature.
```

Comment:
@keefertaylor , this is a breaking change for consumers of `xpring-common-js`, right?

If so, since we're following SemVer, we should probably version to `3.0.0`, since we have a breaking, non backwards compatible change.